packages/docusaurus-plugin-content-blog/src/feed.ts

```diff
@@ -165,16 +166,127 @@ async function defaultCreateFeedItems({
   );
 }
 
+/**
+ * @description addXmlStyleSheet appends a xsl stylesheet to the generated xml feed
+ * @param feedDetails array containing blog feed content and name of file
+ * @param generatePath path where the file would be copied in website
+ */
+async function addXmlStyleSheet(
+  feedDetails: string[],
+  generatePath: string,
+  stylesheet: string,
+) {
+  const DEFAULT_RSS_FEED = 'rss-feed.xslt';
+  const DEFAULT_ATOM_FEED = 'atom-feed.xslt';
+
+  if (!feedDetails[0]) {
+    return feedDetails;
+  }
+
+  if (feedDetails[1] === 'rss.xml') {
+    let xsltLink = `<?xml version="1.0" encoding="utf-8"?><?xml-stylesheet type="text/xsl" href="${DEFAULT_RSS_FEED}"?>`;
```

Comment:
It doens't seems to be useful to have a default value, could be removed to be cleaner

(in fact it seems useful in the case of xslt without style though I don't know the benefits to have xslt without styling)